src/view/channel.js

```diff
@@ -92,6 +93,7 @@ const listStyles = StyleSheet.create({
 const ChannelList = ({ store, channel }) => {
   const {
     computedChannels: channels,
+    channelBalanceOnChainLabel,
```

Comment:
Veeeeery minor but `channelBalanceOnChainLabel` is listed first, but should be moved to last (here and in a few other places in this file), e.g.:
```
channelBalanceOpenLabel,
channelBalanceInactiveLabel,
channelBalancePendingLabel,
channelBalanceClosingLabel,
channelBalanceOnChainLabel,
```
instead of the current order.

Reply:
Fixed!